File: be-api/src/main/java/cd/beapi/dto/response/MedicineResponse.java
package cd.beapi.dto.response;

import java.math.BigDecimal;
import java.time.Instant;

public record MedicineResponse(
        Long id,
        String code,
        String name,
        String unit,
        String description,
        Boolean isActive,
        BigDecimal price,
        Integer quantity,
        String manufacturer,
        String origin,
        Long version,
        Instant createdAt,
        Instant modifiedAt) {
}

File: fe-admin/src/app/core/model/response/medicine-response.ts
export interface MedicineResponse {
    id: number;
    code: string;
    name: string;
    unit: string;
    price:number;
    quantity:number;
    manufacturer:string;
    origin:string;
    description: string;
    isActive: boolean;
    version:number
    createdAt: Date;
    modifiedAt: Date;
}